lib/pxenforcer.js

```diff
@@ -222,7 +222,9 @@ class PxEnforcer {
         const verified = ctx.score < this._config.BLOCKING_SCORE;
         if (res) {
             const setCookie = res.getHeader('Set-Cookie') ? res.getHeader('Set-Cookie') : '';
-            const pxhdCookie = ctx.pxhdServer ? '_pxhd=' + ctx.pxhdServer : '';
+            const expired = `; expires=${new Date(new Date().getTime() + 31536000000).toUTCString()}`;
```

Comment:
you are setting the expired twice? once here and once on line 230?

Reply:
by mistake